src/main/java/org/andersen/homework/model/entity/Ticket.java

```diff
@@ -4,40 +4,118 @@
 import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.Size;
 import java.time.LocalDateTime;
+import java.util.Objects;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Getter;
-import lombok.NoArgsConstructor;
 import lombok.Setter;
-import lombok.ToString;
+import org.andersen.homework.annotation.NullableWarning;
 import org.andersen.homework.model.enums.StadiumSector;
 
 @Getter
 @Setter
-@ToString
-@Builder
-@NoArgsConstructor
 @AllArgsConstructor
 public class Ticket {
 
   @Max(999)
   @Min(0)
-  private Short id;
+  @NullableWarning
+  private final Short id;
 
-  private Float priceInUsd;
+  @NullableWarning
+  private final Float priceInUsd;
 
   @Size(max = 10)
-  private String concertHallName;
+  @NullableWarning
+  private final String concertHallName;
 
   @Max(999)
   @Min(0)
-  private Short eventCode;
+  @NullableWarning
+  private final Short eventCode;
 
-  private Boolean isPromo;
+  @NullableWarning
+  private final Boolean isPromo;
 
+  @NullableWarning
+  private final Float backpackWeightInKg;
```

Comment:
What is the purpose of changing fields to final?)

Reply:
In the 3d point of the task was specified to make only "time" and "stadium sector" fields changeable